docs/how-to/Bar-Memory.rst

```diff
@@ -0,0 +1,102 @@
+
+.. _bar-configuration:
+
+BAR configuration for AMD GPUs
+================================================
+
+For AMD GPUs that have the 44-bit physical address and 48-bit GPU virtual address limit, the BARs are configured as:
+
+.. list-table:: 
+  :widths: 25 25 50
+  :header-rows: 1
+
+  * - BAR Type
+    - Value
+    - Description
+  * - BAR0-1 registers
+    - 64-bit, Prefetchable, GPU memory
+    - 8 GB or 16 GB depending on GPU SKU. Set to less than 2^44 to support P2P access from other GPUs with 44-bit physical address limit. Prefetching enables faster read operation for high-performance computing (HPC) by fetching the contiguous data from the same data source even before requested as an anticipation of a future request.
+  * - BAR2-3 registers
+    - 64-bit, Prefetchable, Doorbell
+    - Set to less than 2^44 to support P2P access from other GPUs with 44-bit physical address limit. As a Doorbell BAR, it indicates to the GPU that a new operation is in its queue to be processed. 
+  * - BAR4 register
+    - Optional
+    - Not a boot device
+  * - BAR5 register
+    - 32-bit, Non-prefetchable, MMIO
+    - Is set to less than 4 GB.
+
+Example of BAR usage on AMD GPUs
+-------------------------------------
+Following is an example configuration of BARs set by the system BIOS on GFX8 GPUs with the 40-bit physical addressing limit: 
+
+.. code:: shell 
+
+  11:00.0 Display controller: Advanced Micro Devices, Inc. [AMD/ATI] Fiji [Radeon R9 FURY / NANO
+  Series] (rev c1)
+
+  Subsystem: Advanced Micro Devices, Inc. [AMD/ATI] Device 0b35
+
+  Flags: bus master, fast devsel, latency 0, IRQ 119
+
+  Memory at bf40000000 (64-bit, prefetchable) [size=256M]
+
+  Memory at bf50000000 (64-bit, prefetchable) [size=2M]
+
+  I/O ports at 3000 [size=256]
+
+  Memory at c7400000 (32-bit, non-prefetchable) [size=256K]
+
+  Expansion ROM at c7440000 [disabled] [size=128K]
+
+Details of the BARs configured in the example are: 
+
+1. GPU Frame Buffer BAR: ``Memory at bf40000000 (64-bit, prefetchable) [size=256M]``
+
+The size of the BAR in the example is 256 MB. In general, it will be the size of the
+GPU memory (typically 4 GB+). Depending upon the physical address limit and generation of AMD GPUs, the BAR can be set below 2^40, 2^44, or 2^48. 
+
+2. Doorbell BAR: ``Memory at bf50000000 (64-bit, prefetchable) [size=2M]``
+
+The size of the BAR should typically be less than 10 MB for this generation of GPUs and has been set to 2 MB in the example. This BAR is placed less than 2^40 to allow peer-to-peer access from other generations of AMD GPUs.
+
+3. I/O BAR: ``I/O ports at 3000 [size=256]``
+
+This is for legacy VGA and boot device support. Since the GPUs used are not connected to a display (VGA devices), this is not a concern even if it is not set up in the SBIOS.
```

Comment:
```suggestion
This is for legacy VGA and boot device support. Because the GPUs used are not connected to a display (VGA devices), this is not a concern, even if it isn't set up in the SBIOS.
```

Reply:
SBIOS - this is correct? not BIOS?

Reply:
system BIOS